src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -697,6 +698,9 @@ public RavenConfiguration CreateDatabaseConfiguration(StringSegment databaseName
             if (databaseRecord.Disabled && ignoreDisabledDatabase == false)
                 throw new DatabaseDisabledException(databaseName + " has been disabled");
 
+            if (databaseRecord.DatabaseState == DatabaseStateStatus.RestoreInProgress)
```

Comment:
Double-checking. We should first check if database is restored and then disabled or other way around is good?

Reply:
done